homeassistant/components/islamic_prayer_times/strings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "config": {
+    "title": "Islamic Prayer Times",
```

Comment:
Just remembered. `title` is now a root level key in the json.

Reply:
I removed it since it is already mentioned in the manifest.

Reply:
Shouldn't this be translatable?

Reply:
I think yes, I will keep it as root key